docs/GETTING_STARTED.md

```diff
@@ -38,7 +38,7 @@ If you don't use PHP 7.4, please change the version in [composer.json](/composer
 
 #### Run the project locally
 
-8. Start your database server and create a new database with the encoding **utf8mb4** and the collation **utf8mb4_unicode_ci** (Be careful: Don't confuse it with *utf8mb4_general_ci*, which is the default for some database management tools. This can lead to hard-to-debug issues later on.)
```

Comment:
I am not sure here. In this video: [https://docs.neos.io/cms/building-your-first-website/local-setup](https://docs.neos.io/cms/building-your-first-website/local-setup)

At minute ~8:05, the author uses `utf8mb4_general_ci`. But the recommendation here is `utf8mb4_unicode_ci`.

Reply:
Yeah, the video was wrong. I changed the video to use the setup tool to create the database.

utf8mb4_unicode_ci is the right choice.